src/main/java/Quag.java

```diff
@@ -0,0 +1,150 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
+public class Quag {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        String userInput;
+        System.out.println("_______________________________________");
+        System.out.println("Hello! I'm QUAG!");
+        System.out.println("What can I do for you today? quag");
+        System.out.println("_______________________________________");
+
+        while (true) {
+            userInput = scanner.nextLine();
+            String command = getCommand(userInput);
+            switch (command) {
+                case "quag":
+                    exitProgram();
+                    return;
+                case "list":
+                    displayList();
+                    break;
+                case "mark":
+                    markTask(userInput);
+                    break;
+                case "unmark":
+                    unmarkTask(userInput);
+                    break;
+                case "todo":
+                    addTodo(userInput);
+                    break;
+                case "deadline":
+                    addDeadline(userInput);
+                    break;
+                case "event":
+                    addEvent(userInput);
+                    break;
+                default:
+                    displayCommandList();
```

Comment:
Effective use of default statement to handle unexpected inputs which would otherwise confuse the user